matcher/service/config/celery_config.py

```diff
@@ -0,0 +1,28 @@
+from kombu import Exchange, Queue
+from kombu.serialization import registry
+
+task_default_queue = "default"
+task_queues = (
+    Queue("matching", routing_key="matcher.service.celery_tasks.run_matching"),
+    Queue(
+        "deployment", routing_key="matcher.service.celery_tasks.run_deployment"
+    ),
+    Queue(
+        "failure", routing_key="matcher.service.celery_tasks.set_error_status"
+    ),
+)
+# celery_imports = 'matcher.service.celery_tasks'
```

Comment:
can you remove the configuration that is not being used?

Reply:
Done.